fluster/test_suite.py

```diff
@@ -521,6 +521,13 @@ def run(self, ctx: Context) -> Optional["TestSuite"]:
             print(f"Skipping decoder {ctx.decoder.name} because it cannot be run")
             return None
 
+        if not os.path.exists(os.path.join(self.resources_dir, self.name)):
+            print(
+                f"Skipping test suite {self.name} because its resources are not available. "
+                f"Please download it first, run `fluster.py --help` for more information"
```

Comment:
This should point to the fluster's `download` command help: `fluster.py download --help`